modules/ckanext-ytp_main/ckanext/ytp/plugin.py

```diff
@@ -1140,6 +1140,7 @@ def helper_linked_user(user, maxlength=0, avatar=20):
         user_name = user.name
         user_displayname = user.display_name
     else:
+        user_name = user
         user = model.User.get(six.text_type(user))
```

Comment:
Could this be changed so that it would not reuse the `user` variable, to make the code itself cleaner ?